src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -398,6 +381,26 @@ export default {
       }
     },
 
+    loadCorrection (filing, section) {
+      if (section) {
+        const item = {
+          type: filing.header.name,
+          filingAuthor: filing.header.certifiedBy,
+          filingDate: filing.correction.correctedFilingDate,
+          filingId: filing.header.filingId,
+          corrFilingId: filing.correction.correctedFilingId,
+          correctedFilingType: this.typeToTitle(filing.correction.correctedFilingType),
```

Comment:
Shouldn't Type be "FilingTypes.xxx"? (where xxx = OTANN etc)

Reply:
Good question, i'll look into this.

Reply:
What I mean is: you're assigning a title to a field named type.

Reply:
Good catch, it actually just needed to be `filing.correction.correctedFilingType` here. Doesn't need a reformat 👍